pipes/auto-pay/pipe.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "auto-pay-pipe",
+  "version": "0.1.0",
+  "fields": [
+
+    {
```

Comment:
this is the old API, we're not using settings like this in the `pipe.json` anymore

Reply:
check https://github.com/mediar-ai/screenpipe/blob/main/pipes/obsidian/src/components/obsidian-settings.tsx